<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectResultLotFrameDialog</class>
 <widget class="QDialog" name="SelectResultLotFrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="EditID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditRemark">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Remark</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IDは何も入れなければ日時が自動的に入る</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCreate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/Qtres/flash_64.PNG</normaloff>:/new/prefix1/Qtres/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>551</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Group definition</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>531</width>
       <height>301</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Group Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Group Lot name</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <attribute name="label">
     <string>History</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetLot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>281</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Day Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remark</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>82</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[G] : Group number </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[F] : Frame number </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
